src/main/kotlin/step2Lotto/domain/LottoNumber.kt

```diff
@@ -0,0 +1,9 @@
+package step2Lotto.domain
+
+class LottoNumber {
+    companion object {
+        private const val MINIMUM_NUMBER = 1
+        private const val MAXIMUM_NUMBER = 45
+        val NUMBERS = (MINIMUM_NUMBER..MAXIMUM_NUMBER).toList()
+    }
+}
```

Comment:
지금 `LottoNumber`는 로또 번호와 관련된 상수만 관리하는 클래스인 것 같아요.
`로또 번호`라는 타입은 도메인에서 중요한 의미를 가지는 만큼 `로또 번호`라는 타입을 새로 정의해보는 건 어떨까요?
```suggestion
class LottoNumber(private val value: Int) {
    init {
         //로또 번호 도메인 검증(1~45 숫자)
    }
    
    companion object {
        // 관련 상수
    }
}
```